charts/values.yaml

```diff
@@ -29,7 +29,7 @@ webhookPKI:
 # This configuration property is a short-lived property. It will be used to toggle between the now deprecated
 # CLI flags and the new OperatorConfiguration (which is the recommended way to configure etcd-druid operator).
 # To preserve backward compatibility the default value for this property is set to false.
-enabledOperatorConfig: false
+enabledOperatorConfig: true
```

Comment:
Please add a release note for this, since it is a breaking change for users of the charts still relying on CLI flags for druid configuration.

@unmarshall FYI. This will not affect the enablement of operator config in gardener/gardener, since gardener uses go objects for the druid deployment rather than the charts. But it's good to keep this in mind.